packages/components/src/__future__/Tabs/_docs/Tabs.spec.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { within, userEvent, expect } from '@storybook/test'
+import { Text } from '~components/Text'
+import { Tab, TabList, TabPanel, Tabs } from '../index'
+
+const meta = {
+  title: 'Components/Tabs/Tabs (Future)/Tests',
+  parameters: {
+    controls: { disable: true },
+  },
+} satisfies Meta
```

Comment:
I'd stick to the pattern for the normal stories so you can utilise the meta for the rest of the stories. The StickerSheet should be the only exception because we give it some special args.

```suggestion
} satisfies Meta<typeof Tabs>
```